bot.js

```diff
@@ -3,17 +3,17 @@ const client = new Discord.Client();
 
 const pasta = ["Spaghetti", "Macaroni", "Ravioli", "Cannelloni", "Linguine", "Lasagne", "Rigatoni", "Tortellini", "Penne", "Fettuccine", "Pappardelle", "Orzi", "Bucatini", "Fusilli", "Vermicelli", "Orecchiette", "Fregula", "Taglierini", "Farfalle", "Manicotti", "Cavatelli", "Paccheri", "Capellini", "Agnolotti", "Strozzapreti", "Cavatappi", "Ziti", "Trofie", "Garganelli", "Conchiglie", "Casarecce", "Rotini", "Pizzoccheri", "Ditalini", "Ptitim", "Gemelli", "Scialatielli", "Cappelletti", "Capelli d’angelo", "Bigoli", "Calamarata", "Mezzelune", "Bavette", "MeMes"]
 
-
 client.on('ready', () => {
   console.log(`Logged in as ${client.user.tag}!`);
 });
 
 client.on('message', msg => {
-  if (msg.author.bot == true) return;
+  if (msg.author.bot)
```

Comment:
Yes, thanks I was thinking about this when I was lying in bed.

Reply:
No problem 👌